lib/screens/start/start_screen.dart

```diff
@@ -30,6 +36,41 @@ class _StartScreenState extends StateWithLifecycle<StartScreen> {
       body: Column(
         mainAxisAlignment: MainAxisAlignment.center,
         children: [
+          const SizedBox(height: 30),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Column(
+                children: [
+                  Observer(
+                    builder: (_) => GeeFutureChild(
+                      loaded: _loadedProjects,
+                      status: _startStore.futureGetProjects.status,
+                    ),
+                  ),
+                  const SizedBox(height: 30),
+                  Center(
+                    child: geeUniversalButton(250, 70, () async {
+                      await GeePopup(context,
+                              content: GeeProjectEditor(
+                                  width: 700,
+                                  heigth: 700,
+                                  startStore: _startStore))
```

Comment:
Z takich drobnych rzeczy, to dodałbym przecinek między nawiasami, wtedy formatowanie ładnie wyrówna tekst, będzie bardziej czytelny